scripts/max-score/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Usage: script.sh <evaluation-directory>
```

Comment:
This should too be added to the Issue

Reply:
Added to https://github.com/symflower/eval-dev-quality/issues/301